bundles/org.openhab.binding.dreamscreen/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dreamscreen"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+
+	<thing-type id="dreamScreen">
+		<label>DreamScreen TV</label>
+		<description>Controls the DreamScreen TV</description>
+
+		<channels>
+			<channel id="power" typeId="power" />
+			<channel id="input" typeId="input" />
+			<channel id="mode" typeId="mode" />
+			<channel id="scene" typeId="scene" />
+			<channel id="color" typeId="color" />
+		</channels>
+
+		<config-description>
+			<parameter name="name" type="text" required="true">
+				<label>Name</label>
+				<description>The name for the DreamScreen</description>
+				<default>DreamScreen 4K</default>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="power">
+		<item-type>Switch</item-type>
+		<label>Power</label>
+		<description>Turns the power to the DreamScreen on/off</description>
+	</channel-type>
+
+	<channel-type id="mode">
+		<item-type>Number</item-type>
```

Comment:
```suggestion
		<item-type>String</item-type>
```

Reply:
Do you mean to have textual command values for mode and then map this to the enum values with valueOf()?

Reply:
Yes something like `<option value="video">Video</option>`. and than you can use them with `valueOf` (I put the value in lowercase, as that would be my personal preference, but requires an additional `toUppercase` before passing to `valueOf`.

The change suggestion specific here it that channels with options must be of item-type `String` and can't be `Number`.

Reply:
changed